src/main/java/com/solace/samples/jcsmp/snippets/HowToConfigureSessionUsingOpenIdConnect.java

```diff
@@ -16,61 +16,117 @@
 
 package com.solace.samples.jcsmp.snippets;
 
-import com.solacesystems.jcsmp.JCSMPProperties;
+import com.solacesystems.jcsmp.*;

```

Comment:
did best practices change? shouldn't we be listing out the individual packages?

Reply:
yes, that is the best practice, but at somestage the IntelliJ code intelligence switches from multiple classes in the same package structure to * imports.